src/api/ledger/accountinfo.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const utils = require('./utils');
+const removeUndefined = require('./parse/utils').removeUndefined;
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+
+function formatAccountInfo(response) {
+  const data = response.account_data;
+  return removeUndefined({
+    sequence: data.Sequence,
+    xrpBalance: utils.common.dropsToXrp(data.Balance),
```

Comment:
I'll bring this question up in the feedback seminar, which I'll probably schedule for next week. Also, send me an email if you can think of any other points that you think I should bring up about the API design.